Formula/orocos-kdl.rb

```diff
@@ -0,0 +1,33 @@
+class OrocosKdl < Formula
+  desc "Orocos Kinematics and Dynamics C++ library"
+  homepage "http://www.orocos.org/kdl"
+  url "https://github.com/orocos/orocos_kinematics_dynamics/archive/v1.3.1.tar.gz"
+  sha256 "aff361d2b4e2c6d30ae959308a124022eeef5dc5bea2ce779900f9b36b0537bd"
+
+  depends_on "cmake" => :build
+  depends_on "eigen@3.2"
```

Comment:
We rarely permit a dependency on a versioned `@` formula for new formulae. Is this unable to use `depends_on "eigen"` still?

Reply:
Fixed.

Reply:
Oh and clearly I'm going to learn something new with each PR subjected to `brew audit --ilovezfs`.